app.rb

```diff
@@ -55,11 +55,13 @@ class App < Sinatra::Base
       :dominant_language => metrics["dominant_language"],
       :is_vendored_framework => metrics["is_vendored_framework"],
       :builds_independently => metrics["builds_independently"],
+      :spm_support => metrics["spm_support"],
     }
 
     github_stats = github_pod_metrics.where(github_pod_metrics[:pod_id] => pod.id).first
+    cocoapods_stats = stats_metrics.where(pod_id: pod.id).first
     owners = owners_pods.outer_join(:owners).on(:owner_id => :id).where(:pod_id => pod.id)
-    data[:quality_estimate] = QualityModifiers.new.generate(data, github_stats, owners)
+    data[:quality_estimate] = QualityModifiers.new.generate(data, github_stats, cocoapods_stats, cocoapods_stats, owners)
```

Comment:
It shouldn't, yeah, I've fixed this locally, got some more QIs to add before the next PR